addons/medical_gui/stringtable.xml

```diff
@@ -3,12 +3,15 @@
     <Package name="Medical GUI">
         <Key ID="STR_ACE_Medical_GUI_Category">
             <English>ACE Medical GUI</English>
+            <German>ACE Sanitäter Oberfläche</German>
         </Key>
         <Key ID="STR_ACE_Medical_GUI_EnableActions_DisplayName">
             <English>Enable Medical Actions</English>
+            <German>Aktiviere Sanitäter Aktionen</German>
```

Comment:
Kein Leerzeichen zwischen Sanitäter und Aktionen. Entweder zusammenschreiben oder Bindestrich.

Reply:
Sanitätsaktionen. I think I already commented that.